examples/metrics/projection.py

```diff
@@ -0,0 +1,382 @@
+#!/usr/bin/env python3
+# Copyright (c) 2024 Intel Corporation
+# SPDX-License-Identifier: 0BSD
+"""
+Understanding Data Projection
+=============================
+
+Projecting data onto P3 definitions.
+
+The P3 Analysis Library expects data to be prepared in a specific
+:ref:`format <performance_data>`. This format was inspired by the
+:ref:`terminology <terminology>` first introduced in "`Implications of a Metric
+for Performance Portability`_":
+
+.. _Implications of a Metric for Performance Portability:
+   https://doi.org/10.1016/j.future.2017.08.007
+
+**Problem**
+  A task with a pass/fail metric for which quantitative performance may be
+  measured.
+
+**Application**
+  Software capable of solving a *problem* with measurable correctness and
+  performance.
+
+**Platform**
+  A collection of software and hardware on which an *application* may run a
```

Comment:
Empasize hardware _and_ software

Reply:
Done in a7b01bbe448c85841f12c30bfe2a83e54eb1f2e0.